frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -349,6 +349,20 @@ const QueueItem = ({
     return false;
   };
 
+  const isFluOnlyDevice = (deviceId: string) => {
+    if (devicesMap.has(deviceId)) {
+      return (
+        devicesMap
+          .get(deviceId)!
+          .supportedDiseaseTestPerformed.filter(
+            (disease) =>
+              disease.supportedDisease.name === MULTIPLEX_DISEASES.COVID_19
+          ).length === 0
```

Comment:
Can we change the logic on this to match the title, as written, its a `isNotACovidDevice` function, not `isFluOnlyDevice`. This logic will break once you add support to any new multiplex disease such as RSV.